pdns/dnsdistdist/docs/guides/dynblocks.rst

```diff
@@ -81,6 +81,6 @@ action is applied.
 .. code-block:: lua
 
   local dbr = dynBlockRulesGroup()
-  -- generate a warning above 100 qps for 10s, and start dropping incoming queries above 300 qps for 10s
+  -- generate a warning above 100 qps for 10s, and start dropping incoming queries above 300 qps for 60s
```

Comment:
The text is misleading, we indeed:
- generate a warning if we receive more than 100 qps during 10s ;
- drop queries for the next 60s if we receive more than 300 qps during 10s.

Perhaps it's only me but I'm afraid that the proposed version is as misleading as the existing one?

Reply:
please check new version